packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1106,4 +1106,67 @@
       <protected>false</protected>
       <shortDescription>Opens in the browser tab</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ComponentError</fullName>
+        <value>{{0}} component error</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>c-quantic-pager component error</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_LookAtDeveloperConsole</fullName>
+      <value>Look at the developer console for more information.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Look at the developer console</shortDescription>
+    </labels>
+        <labels>
+      <fullName>quantic_InvalidMaxNumberOfDocumentSuggestions</fullName>
+      <value>"{{0}}" is an invalid maximum number of document suggestions. An integer greater than 0 was expected.</value>
```

Comment:
@MehdiMitiche Maybe I'm simplifying too much, but we could go straight to the solution, as 0 is obviously not valid as a maximum?

"The maximum number of document suggestions must be greater than 0."

Reply:
Totally agree!